avm-transpiler/src/transpile.rs

```diff
@@ -822,7 +822,8 @@ fn handle_black_box_function(avm_instrs: &mut Vec<AvmInstruction>, operation: &B
                 ..Default::default()
             });
         }
-        BlackBoxOp::ToRadix { input, radix, output } => {
+        // We ignore the output bits flag since we represent bits as bytes
```

Comment:
Ok IIUC, this works ok right now but when we implement u1s, we have to either support "output bits" in the opcode, or (possibly temporarily) add a cast to u1.

This is probably ok but tagging @IlyasRidhuan @jeanmon